config/locales/en.yml

```diff
@@ -210,3 +210,4 @@ en:
     landing_page: "Apply for an early-career payment"
     questions:
       nqt_in_academic_year_after_itt: "Did you do your NQT the academic year after your ITT?"
+      poor_performance: "Are you currently subject to action for poor performance?"
```

Comment:
@dbrunnellAD this needs a new line after the last entry. Thats why it has that no smoking sign at the end in red. Could you fix and push up that chnage please.

Otherwise the view Looks Good to Me (my LGTM version) and I will pick-up to do the wiring-up. Thanks